api.js

```diff
@@ -36,91 +37,92 @@ $(function() {
         }
     });
 
-    $("#submit").on("click", function(){
 
-        $( "#clear" ).click(function() {
-            $( "#output" ).empty();
-            
+    $("#submit").on("click", function () {
+
+        $("#clear").click(function () {
+            $("#output").empty();
         });
-    
-        var artist = $("textarea").val();
-        var queryURL = "https://en.wikipedia.org/w/api.php?action=opensearch&search="+ artist +"&format=json&callback=?";
```

Comment:
remove this **var** and **queryURL** the one bellow is the one we will be working with